content/issue-1/weihanglo.md

```diff
@@ -0,0 +1,131 @@
+
+Back to the day I really loves of the concept behind open source communities. It is not really the classic "free as freedom" open source idea. It's more like "sharing is caring". I love one people sharing their ideas their knowledge and everything. Without thinking too much like "I give this out and wait for returns", the generosity did make me feel wholesome. I have received a lot of helps from the community when I started my career as a software engineer, so I want to give it back to the community and let sharing continue.
+
+I was a web frontend developer in 2017, seeking opportunities to sharpen my skills outside web development. I'd like to learn more on system programming and compiled languages. There were a couple of choice: Golang from Google and Rust from Mozilla. I took a glimpse at Golang, though the development of Golang itself was not really open-source minded at that time. Instead, I bet on Rust, not because its safety guarantee, but the openness of the community. And I trust Mozilla as well, regarding how they operation open source communities.
+
+I attended the Mozilla conference in Taipei and spoke to many people, including someone who had written a lot of content on Rust and is now an author of two Rust books. During the conference, I learned about the Servo engine in Rust. I also met others interested in learning Rust. The community was supportive and friendly, so we organized a reading club to read through the entire Rust book. That's how I got started with Rust.
+
+What inspires me to stick with Rust is the community and the shared belief that Rust will succeed. I keep learning and advocating Rust because it is a pragmatic and safe programming language, acting as a bridge between paradigms. The ownership and lifetime model in Rust eliminates common programming errors, making me feel safe when writing code. Rust is like an mother from East Asia who wants to control your life, but also nurtures and teaches you. This is why I keep learning and advocating Rust. I cannot program anything without Rust. I am already spoiled by Rust.
+"""
+```
+
+# The story behind joining the Cargo team
+
+**Can you share the story of how you joined the Cargo team? What was the process like and what attracted you to the opportunity?**
+
+```quote
+author = "Weihang Lo"
+content = """
+To be honest, there was no a clear path to become an official member of the Cargo team. It's more like a long process of earning the trust of existing members. I've been contributing to the project since 2019 during the lockdown and my first pull requests is simple. It was a fix related to lockfile comparsion. I removed some unnecessary allocation of the comparison function and switched from string comparison to iterator comparison, which was encouraged by a kind maintainer Jacob (Eh2406). He recognized my efforts and I felt more welcomed with the project.
```

Comment:
Hope the "first pull requests" is the link :)

Reply:
It's this one: https://github.com/rust-lang/cargo/pull/8641

BTW, my memory got rusty. I started contributing Cargo since 2020 not 2019, could you perform a `s/2019/2020` for this article?

Reply:
Sure